docs/linux.md

```diff
@@ -75,6 +88,51 @@ Issue the following command from the terminal:
 	OpenSSL* 1.1.1q  05 Jul 2022
 	```
 
+#### Steps to install curl version 7.86 configured with openssl
+
+After installing openssl, proceed with the installation of curl.
+
+1. Pull the tarball:
+	```
+	wget https://github.com/curl/curl/releases/download/curl-7_86_0/curl-7.86.0.tar.gz
+	```
+2. Unpack the tarball with:
+	```
+	tar -zxf curl-7.86.0.tar.gz && cd curl-7.86.0
+	```
+3. Issue the command to configure the curl with previously downloaded openssl 3:
+	```
+	./configure --with-openssl
```

Comment:
also add '--enable-versioned-symbols' to fix error: curl: /usr/local/lib/libcurl.so.4: no version information available (required by curl)